numpy/linalg/lapack_lite/f2c_config.c

```diff
@@ -239,7 +237,6 @@ doublereal dlamch_(char *cmach)
 
 
     if (first) {
-	first = FALSE_;
```

Comment:
See above. I think it is easier to follow setting `first` here, i.e., execute only the first time.